build/script/travis-build.sh

```diff
@@ -27,12 +27,24 @@ npm run pack
 
 echo Using neovim path: "$ONI_NEOVIM_PATH"
 
-npm run test:integration
-
 if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+   echo Installing reason CLI tools...
+   npm install -g https://github.com/reasonml/reason-cli/archive/3.0.4-bin-darwin.tar.gz
```

Comment:
You may want to try the newest `npm install -g reason-cli@3.1.0-darwin`
(Faster download, more updated merlin).

Reply:
Excellent! Thanks @jordwalke - testing it out now.